data/data.go

```diff
@@ -0,0 +1,422 @@
+// Package data implements sql operations over the NoName e.V. website-db.
+package data
+
+import (
+	"database/sql"
+	"errors"
+	"flag"
+	"fmt"
+	"net/url"
+	"strconv"
+	"strings"
+	"time"
+)
+
+var (
+	driver  = flag.String("driver", "postgres", "Der benutzte sql-Treiber")
+	connect = flag.String("connect", "dbname=nnev user=anon host=/var/run/postgresql sslmode=disable", "Die Verbindusgsspezifikation")
```

Comment:
I think we should be consistent with the row length. Either be as long as it needs to be or not using 80 chars as a line-break-reference (compare L.25, declaration of the Querier interface, which I somehow fail to link properly).